### DISCUSSION TITLE
申请退款邮箱被阻请求协助！@Mike777Khoo

### DISCUSSION THREAD

**Discussion by vincent122287** [getlantern/lantern-forum-cn#416] [General]
@Mike777Khoo发送退款申请邮件被阻，ios版本蓝灯和window版本蓝灯频繁断网，申请退款
账号：344485316@qq.com
![微信图片_20240409113118](https://github.com/getlantern/lantern-forum-cn/assets/150009674/e207c4e6-fe75-4bae-8b78-38f56738d349)

**Comment by vincent122287** [comment]
![image](https://github.com/getlantern/lantern-forum-cn/assets/150009674/fc64c563-01f5-4150-b419-c151eba97c7b)
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@vincent122287 好的. 我另外帮您处理. 我会帮您直接提交给付款团队的负责人.

需要您等 7-10 天. 不好意思最近客服那里太忙了. 我们会尽快!

**Reply by vincent122287** [reply]
谢谢 希望有消息了尽快在这条帖子下反馈

**Reply by vincent122287** [reply]
@Mike777Khoo 还未收到退款 请催办

**Comment by vincent122287** [comment]
未收到退款 @Mike777Khoo 请催办
